### PR TITLE
fix and enhance the clusterset part

### PR DESCRIPTION
**Pull request by zhiweiyin318**
Signed-off-by: Zhiwei Yin <zyin@redhat.com>

part of task in https://github.com/stolostron/backlog/issues/27039

### CODE REVIEW

**Review by ldpliu — APPROVED** [review]

**Comment by swopebe** [comment]
Communication between the foundation writer and developers is very important, we have a managed cluster set doc issue: https://github.com/stolostron/backlog/issues/26958

@cadawson is aware of this issue but we gave it to Oliver for the rework. Tagging the issue to for awareness.

Please open doc issues, writers should use channels, scrums, issues to communicate with developers so we have adequate reviews and knowledge of doc changes.

File: multicluster_engine/cluster_lifecycle/managedclustersets_intro.adoc [hunk 1]
```diff
@@ -5,7 +5,7 @@ A `ManagedClusterSet` is a group of managed clusters. With a managed cluster set
 
 Each managed cluster must be a member of a managed cluster set. When you install the hub cluster, a default `ManagedClusterSet` resource is created called `default`. All managed clusters that are not specifically assigned to a managed cluster set are automatically assigned to the `default` managed cluster set. To ensure that the default managed cluster set is always available, you cannot delete or update the `default` managed cluster set.
 
-**Note:** Cluster pools that are not specifically added to a managed cluster set are not added to the default `ManagedClusterSet` resource. After a managed cluster is claimed from a cluster pool, it is added to the default `ManagedClusterSet` resource if it is not specifically added to another managed cluster set.
+**Note:** Cluster pools that are not specifically added to a managed cluster set are not added to the default `ManagedClusterSet` resource. After a managed cluster is claimed from the cluster pool, the managed cluster will be added to the default `ManagedClusterSet`.
```
Comment (swopebe) [inline multicluster_engine/cluster_lifecycle/managedclustersets_intro.adoc]: is added

File: multicluster_engine/cluster_lifecycle/managedclustersets_remove_cluster.adoc [hunk 2]
```diff
@@ -10,17 +10,15 @@ You might want to remove a managed cluster from a managed cluster set to move it
 
 Complete the following steps to remove a cluster from a managed cluster set by using the console:
 
-. If you just created your managed cluster set, select *Manage resource assignments* to go directly to the _Manage resource assignments_ page. Continue with step 5 of this procedure. 
-
-. If your cluster already exists, access the cluster page by selecting *Infrastructure* > *Clusters* in the main navigation and ensure that the _Cluster sets_ tab is selected. 
+. Access the cluster page by selecting *Infrastructure* > *Clusters* in the main navigation and ensure that the _Cluster sets_ tab is selected.
```
Comment (swopebe) [inline multicluster_engine/cluster_lifecycle/managedclustersets_remove_cluster.adoc]: Click *Infrastructure* > *Clusters* ..

File: multicluster_engine/cluster_lifecycle/managedclustersets_remove_cluster.adoc [hunk 3]
```diff
@@ -10,17 +10,15 @@ You might want to remove a managed cluster from a managed cluster set to move it
...
 
 . Select the name of the cluster set that you want to remove from the managed cluster set to view the cluster set details.
 
 . Select *Actions* > *Manage resource assignments*.
 
-. On the _Manage resource assignments_ page, select the checkbox for the resources that you want to remove from the cluster set. 
+. On the _Manage resource assignments_ page, remove the checkbox for the resources that you want to remove from the cluster set. 
 +
-This step removes a resource that is already a member of the cluster set, or adds a resource that is not already a member of the cluster set. You can see if the resource is already a member of a cluster set by viewing the details of the managed cluster.   
+This step removes a resource that is already a member of the cluster set. You can see if the resource is already a member of a cluster set by viewing the details of the managed cluster.
```
Comment (swopebe) [inline multicluster_engine/cluster_lifecycle/managedclustersets_remove_cluster.adoc:22]: Please ensure that is this is considered new function that it is mentioned in What's new....if we are just catching up, we can skip that. Thanks for the changes. Peer review done.

**Review by swopebe — APPROVED** [review]
Peer review, still need main writer.

File: multicluster_engine/cluster_lifecycle/managedclustersets_remove_cluster.adoc (hunk 3 above)
Comment (zhiweiyin318) [reply multicluster_engine/cluster_lifecycle/managedclustersets_remove_cluster.adoc:22]: this is not a new function, the remove steps in the old doc are incorrect.

**Review by cadawson — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**